drupal/profiles/favrskov/modules/custom/favrskov_helper/favrskov_helper.install

```diff
@@ -701,3 +701,110 @@ function favrskov_helper_update_7037() {
+  $field_text_summary = field_read_fields(array('type' => 'text_with_summary'));
+  $field_long_text = field_read_fields(array('type' => 'text_long'));
+  $fields = array_merge($field_long_text, $field_text_summary);
+  $fields_name = array_keys($fields);
+  $result_value = array();
+  foreach ($fields_name as $name) {
+    $data_result = db_select('field_data_' . $name, 'fd')
+      ->fields('fd', array('entity_type', 'entity_id'))
+      ->condition($name . '_value', '%' . db_like('media-image') . '%', 'LIKE');
+    $data_result->addField('fd', $name . '_value', 'value');
+    $data_result = $data_result->execute()->fetchAll();
+    $result_value = array_merge($result_value, array_values($data_result));
+  }
+
+  $files_usage = array();
+  foreach ($result_value as $fields) {
+    if (empty($fields)) {
+      continue;
+    }
+
+    $file_pattern = preg_match_all('/<span.*class="file media-image.*<\/span>/', $fields->value, $file_match);
+    $image_patern = preg_match_all('/<img.*class="media-image.*\/>/', $fields->value, $image_match);
+
+    if ($image_match) {
+      foreach ($image_match[0] as $key => $img) {
+        $image_url = preg_match('/img__fid__(\d+)/', $img, $url);
+        if ($image_url == 0) {
+          continue;
+        }
+        $std_file = new stdClass();
+        $std_file->fid = $url[1];
+        $std_file->entity_id = $fields->entity_id;
+        $std_file->entity_type = $fields->entity_type;
+        preg_match('/src="([^"]+)"/', $img, $src);
+
+        if (isset($src[1])) {
+          $last_part = explode('/', $src[1]);
+          $std_file->link = end($last_part);
+        }
+
+        $files_usage[] = $std_file;
+      }
+    }
+    if ($file_match) {
+      foreach ($file_match[0] as $key => $file) {
+        $file_url = preg_match('/img__fid__(\d+)/', $file, $url);
+        if ($file_url == 0) {
+          continue;
+        }
+        $std_file = new stdClass();
+        $std_file->fid = $url[1];
+        $std_file->entity_id = $fields->entity_id;
+        $std_file->entity_type = $fields->entity_type;
+        preg_match('/href="([^"]+)"/', $file, $href);
+        if (isset($href[1])) {
+          $last_part = explode('/', $href[1]);
+          $std_file->link = end($last_part);
+        }
+        $files_usage[] = $std_file;
+      }
+    }
+  }
+
+  $location = 'temporary://node_errors.txt';
+  $fp = fopen($location,"w");
+
+  foreach ($files_usage as $file) {
+    $error = FALSE;
+    if (empty($file->fid) || empty($file->link)) {
+      continue;
+    }
+
+    $fid = db_select('file_managed', 'fm')
+      ->fields('fm', array('fid', 'uri'))
+      ->condition('fid', $file->fid, '=')
+      ->execute()
+      ->fetchAll();
+    if (empty($fid[0])) {
+      $output = sprintf('Doesn\'t exist -> Node entity: %s, node id: %s, file id: %s', $file->entity_type, $file->entity_id, $file->fid);
+      $error = TRUE;
+      fwrite($fp, $output . PHP_EOL);
+    }
+
+    if (isset($file->link) && !empty($fid[0])) {
```

Comment:
You can combine checks
if (empty($fid[0])) {
...
}
elseif (isset($file->link)) {
...
}